apps/web/src/pages/quick-start/components/demo-flow/nodes/NodeButton.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { UnstyledButton } from '@mantine/core';
+
+import { useStyles } from '../../../../../design-system/template-button/TemplateButton.styles';
+import { colors, Text } from '../../../../../design-system';
+
+export function NodeButton({
+  data,
+  id,
+  Handlers,
+  Icon,
+  ActionItem,
+}: {
+  data: any;
+  id: string;
+  Handlers: React.FC<any>;
+  Icon: React.FC<any>;
+  ActionItem?: React.FC<any>;
+}) {
+  const { theme } = useStyles();
+
+  return (
+    <>
+      <Button
+        data-test-id={`data-test-id-${data.label}`}
+        sx={{
+          backgroundColor: theme.colorScheme === 'dark' ? colors.B17 : colors.white,
+        }}
+      >
+        <ButtonContent>
+          <ButtonLeftContent>
+            <Icon style={{ marginRight: '15px' }} />
+            <Text weight={'bold'}>{data.label} </Text>
+          </ButtonLeftContent>
+          {ActionItem ? <ActionItem /> : null}
```

Comment:
This part is a hot placeholder in the action section on the right side on the button I left for the playground stage, @LetItRock  you can use it as well, I am still not sure I will need it on my page.

https://www.figma.com/file/hlsUxjq2kxUU7aVBf2QyGu?node-id=409:41745#384032833